### PR TITLE
Rating Refresh issue

### PR DESCRIPTION
**Pull request by shirishgoyal**
Backend improvements
Frontend refresh issue resolved

image

### CODE REVIEW

File: crowdsourcing/viewsets/rating.py [hunk 1]
```diff
@@ -17,19 +17,20 @@ class WorkerRequesterRatingViewset(viewsets.ModelViewSet):
...
-            return Response({'status': 'Rating created'})
+            wrr = wrr_serializer.create(origin=request.user.userprofile)
+            wrr_serializer = WorkerRequesterRatingSerializer(instance=wrr)
+            return Response(wrr_serializer.data, status=status.HTTP_201_CREATED)
         else:
             return Response(wrr_serializer.errors,
                             status=status.HTTP_400_BAD_REQUEST)
 
-    def update(self, request, pk=None):
+    def update(self, request, *args, **kwargs):
```
Comment (dmorina) [inline crowdsourcing/viewsets/rating.py]: ;)

File: static/js/dashboard/controllers/dashboard.controller.js [hunk 2]
```diff
@@ -1,207 +1,204 @@
...
+                    entry.reviewType = reviewType;
+                }, function error(resp) {
+                    $mdToast.showSimple('Could not update rating.');
+                }).finally(function () {
+
+                });
+            } else {
+                RankingService.submitRating(rating, entry).then(function success(resp) {
+                    entry = resp;
+                    entry.reviewType = reviewType;
```
Comment (dmorina) [inline static/js/dashboard/controllers/dashboard.controller.js]: review type cannot change

**Comment by dmorina** [comment]
see comments, otherwise lgtm

**Comment by dmorina** [comment]
profile picture would be better to be removed here as we dont have that functionality yet, I know it was there before

**Comment by aginzberg** [comment]
all the new stuff looks good. i know my screen is narrow but do you think we can give the checks a bit more space- i have to make my browser really wide to look like your image screen shot 2015-09-03 at 12 36 21 pm

**Comment by nistala** [comment]
- Plus and minus next to checks are not aligned correctly.
- If I click check plus and the action fails, incorrect action is active

**Comment by dmorina** [comment]
screen shot 2015-09-03 at 21 01 39
functionality wise it seems to work well for me

**Comment by nistala** [comment]
@dmorina the request has to fail as I mentioned in #475

**Comment by shirishgoyal** [comment]
I specifically aligned + and - signs like that. In case those signs need to be below check sign, please let me know.

There is no change happening if action fails. So probably I am missing the logic how it is impacting function.

**Comment by dmorina** [comment]
could you post a screenshot of what errors you get? @nistala

**Comment by nistala** [comment]
screenshot from 2015-09-03 12 06 30

**Comment by mbernst** [comment]
I would recommend going with the standard alignments of the + and -:
http://sweetclipart.com/multisite/sweetclipart/files/grade_marks.png

**Comment by shirishgoyal** [comment]
Feedback incorporated for rating updates

Icon alignment corrected as shown below
image

**Comment by dmorina** [comment]
lgtm, resolve conflicts please

**Comment by shirishgoyal** [comment]
Conflicts Resolved

**Comment by aginzberg** [comment]
lgtm

**Pull request merged**